CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
-# dbt_recurly_source v0.UPDATE.UPDATE
+# dbt_recurly_source v0.3.0
+[PR #16](https://github.com/fivetran/dbt_recurly_source/pull/16) introduces the following updates:
 
- ## Under the Hood:
+## Breaking Changes
+- Removed unused fields from `stg_recurly__subscription_history`:
+  - `converted_at`
+  - `has_started_with_gift`
+- These fields were removed since they are planned for future deprecation in the Fivetran Connector and are not utilized in the downstream transformation package.
+
+
+[PR #14](https://github.com/fivetran/dbt_recurly_source/pull/14) includes the following updates:
```

Comment:
Is this referring to the Under the Hood changes? Maybe we should add a link to this PR at the end of those sentences rather than have two sentences of this format in the same release notes.

Reply:
Yeah it was a pr from a while ago. Updated!